src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,66 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String DATA_FIELD_SEPARATOR = ",";
+    private static final int OPERATION_TYPE_INDEX = 0;
+    private static final int AMOUNT_INDEX = 1;
+    private static final String SUPPLY_FIELD_NAME = "supply";
+    private static final String BUY_FIELD_NAME = "buy";
+    private static final String RESULT_FIELD_NAME = "result";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        writeToFile(toFileName, processData(readDataFromFile(fromFileName)));
+    }
+
+    private String[] readDataFromFile(String nameFile) {
+        File file = new File(nameFile);
+        StringBuilder stringBuilder = new StringBuilder();
+        try (BufferedReader reader = new BufferedReader(new FileReader(file))) {
+            String value = reader.readLine();
+            while (value != null) {
+                stringBuilder.append(value).append(System.lineSeparator());
+                value = reader.readLine();
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("Cant read file", e);
+        }
+        return stringBuilder.toString().split(System.lineSeparator());
+    }
+
+    private String processData(String[] data) {
```

Comment:
Consider using a more descriptive method name instead of `processData`. For example, `calculateStatistics`. 🤔